src/classes/metadata/pdf.vala

```diff
@@ -195,6 +207,39 @@ namespace pdfpc.Metadata {
                         }
                     }
                 }
+
+                if (this.notes_include != null) {
+                    parse_notes_file();
+
+                    // if we have a notes include option, completly ignore the [notes] section
+                    if (notes_content != null) {
+                        GLib.printerr("Found [notes_include] and [notes] section. Ignoring [notes]\n");
+                    }
+                } else if (notes_content != null) {
+                    notes.parse_lines(notes_content.split("\n"));
+                }
+            } catch (Error e) {
+                GLib.printerr("%s\n", e.message);
+                Process.exit(1);
+            }
+        }
+
+
+        /**
+         * Parse an additonal notes file
+         */
+        void parse_notes_file() {
+            try {
+                string content;
+                string full_notes_include = GLib.Path.build_filename(GLib.Path.get_dirname(this.pdfpc_fname), this.notes_include);
```

Comment:
This will mean that absolute paths won't work from pdfpc files, because the `build_filename` function simply concatenates. Do we want that?